src/Rule/RegexHandler.php

```diff
@@ -27,13 +27,7 @@ public function validate(mixed $value, object $rule, ValidationContext $context)
         $result = new Result();
 
         if (!is_string($value)) {
-            $result->addError(
-                $rule->getIncorrectInputMessage(),
-                [
-                    'attribute' => $context->getAttribute(),
-                    'value' => $value,
```

Comment:
`value` can be useful for custom messages.

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006452279

Reply:
https://github.com/yiisoft/validator/pull/352#discussion_r1006471638